core/loadpoint.go

```diff
@@ -445,9 +445,6 @@ func (lp *Loadpoint) evChargeStartHandler() {
 func (lp *Loadpoint) evChargeStopHandler() {
 	lp.log.INFO.Println("stop charging <-")
```

Comment:
Hier dann besser auch
```suggestion
	lp.log.INFO.Println("vehicle stopped charging <-")
```
?

Reply:
Denke nicht- warum das passiert ist unklar.